### PR TITLE
FenwickTree in C#(jainaman224#2173)

### LINKED ISSUE #2173: Fenwick Tree
- [x] C
- [x] C++
- [ ] C# - @ritish099
- [ ] CoffeeScript
- [x] Dart - @Rukmini-Meda
- [ ] Go
- [x] Java
- [x] JS
- [x] Kotlin - @JasbirCodeSpace
- [x] Php - @goyalmuskan
- [x] Python
- [ ] Readme - @AJAYSHARMA2012
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by IPSITA2000**
Fixes #2173

#### Checklist:
- [ *] 4 space indentation.
- [ *] Coding conventions are followed.
- [ *] Input is taken dynamically.
- [ *] Sample Input / Output is added at the end of file.
- [ *] Logic Documentation (Comments).
- [ *] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Changing into dynamic Input 
- Adding issue number

#### Languages Used:
- C#
- 

#### Files Added:
- Fenwick_Tree.cs
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by SakshiShreya** [comment]
Please fill the PR template properly and also mention the issue number after `Fixes #`

**Comment by IPSITA2000** [comment]
Okay , I will do it .Thanks for informing me.

**Comment by IPSITA2000** [comment]
@SakshiShreya  I have made the necessary changes . Please review it !

File: Fenwick_Tree/Fenwick_Tree.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          
+        // Update BIT for above change in arr[]  
+        updateBIT(n, 3, 6);  
+  
+        // Find sum after the value is updated  
+        Console.WriteLine("Sum of elements in arr[0..5]"+  
+                    " after update is " + tree.getSum(5));  
+    }  
+}  
+
```
Comment (somya-kapoor) [inline Fenwick_Tree/Fenwick_Tree.cs]: Add sample input and output

Comment (IPSITA2000) [reply Fenwick_Tree/Fenwick_Tree.cs]: Ok!

**Comment by somya-kapoor** [comment]
@IPSITA2000 Are you working on it?

**Comment by IPSITA2000** [comment]
@somya-kapoor  I am extremely sorry for the delay . I have done added the sample input and output . Please review it.

**Comment by somya-kapoor** [comment]
Copied from Gfg

**Comment by IPSITA2000** [comment]
I am sorry sir , but fenwick tree was  my first program in c# and also on github. when i got this , i found that gfg had the best explanation to the fenwick tree and i went through the code ,analysing it and when i  tried of my own i couldnt think of a new way and ended up writing something way similar to one on gfg . I know i should have not taken the fenwick tree implementation but being totally new , i was unware that i can turn down my request too!
My sincere apologies to you. @somya-kapoor

**Pull request closed without merging**